app/command.go

```diff
@@ -435,8 +435,8 @@ func (a *App) HandleCommandResponsePost(command *model.Command, args *model.Comm
 		post.AddProp("from_webhook", "true")
 	}
 
-	// Do not process text if this is a code block
-	skipSlackParsing := command.Trigger == "code"
+	// Do not process text if this is a code block or response contains "skip_slack_parsing":"true"
+	skipSlackParsing := command.Trigger == "code" || (len(response.SkipSlackParsing) != 0 && response.SkipSlackParsing == "true")
```

Comment:
Would you mind removing the special rule for `code` and update `app/command_code.go` to use `SkipSlackParsing`?

Reply:
done.

Reply:
after botching this PR, i created a new one.

Is there a useful use-case that not all built in commands should be excluded from the Slack-Compatibility layer?